pkg/fqdn/bootstrap/fqdn_bootstraper.go

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
typo in filename: `bootstraper` -> `bootstrapper` (also for test)

Reply:
thanks, fixed